docs/ClusterSetup.md

```diff
@@ -221,6 +224,7 @@ zookeeperServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.u
 globalZookeeperServers=zk1.us-west.example.com:2184,zk2.us-west.example.com:2184,zk3.us-west.example.com:2184
 
 clusterName=us-west
+statusFilePath=broker-status-file
```

Comment:
I'd swear I setup the test env with the above step and this setting was not preventing the broker from functioning. I need some time to look into it again